src/SDKs/CognitiveServices/dataPlane/Vision/CustomVision/Prediction/Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction.csproj

```diff
@@ -6,14 +6,15 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services CustomVision Prediction APIs.</Description>
-    <VersionPrefix>0.9.0-preview</VersionPrefix>
+    <Version>0.10.0-preview</Version>
```

Comment:
There is a diff here with the private repo PR.
Also, PackageReleaseNotes indicate this is not a preview release but the Version does, please verify the intent here

Reply:
Yes, I was asked to modify the version and update the preview outside of the private pr. I can send you the info so you can validate.